code/src/java/pcgen/cdom/content/ProcessCalculation.java

```diff
@@ -62,38 +62,25 @@ public ProcessCalculation(T object, BasicCalculation<T> calc,
 		this.formatManager = fmtManager;
 	}
 
-	/**
-	 * {@inheritDoc}
-	 */
 	@Override
 	public T process(EvaluationManager evalManager)
 	{
-		@SuppressWarnings("unchecked")
```

Comment:
Please leave these.  We are checking code with various tools like PMD and Checkstyle (among others) and these suppresswarnings entries serve a purpose in indicating that the code has been reviewed and is not a valid warning.